addons/headless/CfgEden.hpp

```diff
@@ -0,0 +1,35 @@
+class Cfg3DEN {
+    class Object {
+        class AttributeCategories {
+            class ace_attributes {
+                class Attributes {
+                    class GVAR(blacklist) {
+                        displayName = "Headless Blacklist";
+                        tooltip = "If blacklisted this vehicle will not be transfered to headless clients.";
+                        property = QGVAR(blacklist);
+                        control = "Checkbox";
+                        condition = "objectVehicle";
```

Comment:
I believe this will allow it for all vehicles, but it should only be visible on actual infantry units.